tests/openapi/openapi_integration/test_sparse_idf.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+
+from .helpers.collection_setup import drop_collection
+from .helpers.helpers import request_with_validation
+
+collection_name = 'test_sparse_idf'
+
+
+@pytest.fixture(autouse=True)
+def setup():
+    idf_collection_setup(collection_name=collection_name)
+    yield
+    # drop_collection(collection_name=collection_name)
```

Comment:
This line is commented. Is this intended?

Reply:
forgot after debugging, thanks!